.github/workflows/add-to-project.yml

```diff
@@ -11,7 +11,14 @@ jobs:
     name: Add issue to project
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/add-to-project@v0.5.0
+      - name: Generate GitHub token
+        uses: tibdex/github-app-token@v2
+        id: generate-github-token
+        with:
+          app_id: ${{ secrets.GH_APP_ID }}
+          private_key: ${{ secrets.GH_APP_PRIVATE_KEY }}
+      - name: Add to project
+        uses: actions/add-to-project@v0.5.0
         with:
           project-url: https://github.com/orgs/camunda/projects/33
```

Comment:
Let's take the chance and set that as a GHA env var.

Reply:
done.